### PR TITLE
README: Explain how to execute website changes

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Eventually we may want to make this automatic, but let's at least clearly document, immediately, what to do.

Signed-off-by: David A. Wheeler <dwheeler@dwheeler.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Metamath website scripts
 
-This repository has various scripts to set up a Metamath website.
+This repository has various scripts to set up or reconfigure a Metamath website.
+
+If you just want to change the configuration of the existing Metamath website,
+change this repo's "main" branch. Then log in with `ssh root@us.metamath.org`
```
Comment (tirix) [inline README.md]: Is that useful for you only?
Maybe mention who is the current maintainer and that anyone who wants to help shall contact you for access?

Comment (david-a-wheeler) [reply README.md]: Mario too, and anyone else with the permissions.

You're right, I can mention who has those permissions.

Comment (digama0) [reply README.md]: By the way, I am willing to serve as a conduit for people who want to make website changes. Make a PR here and/or a thread on the mailing list, and we can explore new functionality or changes to existing things.

**Comment by david-a-wheeler** [comment]
Me too, of course. The reason to put this into GitHub was to make it easier to propose, review, and track changes.

File: README.md (hunk 1 above)
Comment (david-a-wheeler) [reply README.md]: I tweaked the text further and I *think* I resolve this. I'm going to mark the conversation resolved, if it's not resolved or you see other problems, please re-raise an issue!

**Comment by tirix** [comment]
Yes, perfect!

**Review by tirix — APPROVED** [review]

**Pull request merged**